src/vaishnaviVJun20_Collection/Retrieve.java

```diff
@@ -0,0 +1,29 @@
+package vaishnaviVJun20_Collection;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/*Assignment 35:  Retrieve all unique names from given array.
+
+String[] name1 = {"Palak", "Viresh", "Yash", "Aavruti"};
+String[] name2 = {"Deavina","Palak","Viresh", "Nikhil"};
+output : [Palak, Viresh, Yash, Aavruti,Deavina,Nikhil] */
+
+public class Retrieve {
+	
+	void retrieveAllUniqueNames(ArrayList<String> name1, ArrayList<String> name2) {
+		name2.removeAll(name1);
+		name1.addAll(name2);
+		System.out.println("Output: " +name1);
+	}
+
+	public static void main(String[] args) {
+		String[] name1 = {"Palak", "Viresh", "Yash", "Aavruti"};
+		String[] name2 = {"Deavina","Palak","Viresh", "Nikhil"};
+		ArrayList<String> arrList1 = new ArrayList<String>(Arrays.asList(name1));
```

Comment:
I am happy, you could figure out arrayList2.removeAll(arrayList1) is required and asList method returns List which could not be modified, hence we need to create new ArrayList object and call asList method from constructor.
Code looks good to me.